packages/inertia-react/package.json

```diff
@@ -35,7 +35,7 @@
     "eslint": "^7.0.0",
     "eslint-plugin-react": "^7.14.3",
     "eslint-plugin-react-hooks": "^4.0.0",
-    "microbundle": "^0.12.0",
+    "microbundle": "^0.14.0",
```

Comment:
Upgrade microbundle to ^0.14.0, will update typescript to ^4.1.3.

(microbundle@^0.12.0 dependents the typescript ^3.9)